modules/custom/social_lazy_loading/social_lazy_loading.module

```diff
@@ -4,3 +4,16 @@
  * @file
  * Provides basic Blazy integration for lazy loading and multi-serving images.
  */
+
+/**
+ * Implements hook_preprocess().
+ */
+function social_lazy_loading_preprocess_node(&$variables) {
+  /** @var \Drupal\node\Entity\Node $node */
+  $node = $variables['node'];
+  if ($node->getType() === 'landing_page') {
+    // Add lazy loading to landing pages.
+    // Make sure it will be reactivated.
+    $variables['content']['#attached']['library'][] = 'social_lazy_loading/lazy_load_reactivate';
+  }
+}
```

Comment:
Landing page already has a library definition that's attached. Perhaps we should implement a `library_alter` and add our library/CSS there.

This way we don't introduce a hook that's called for every node :D